Lib/_pyio.py

```diff
@@ -584,10 +589,11 @@ def backreadline(self, size=-1):
             if b == b"\n":
                 eol = b
                 break
-        if eol is not None:
-            # reverse the characters in the line, except the new line character
-            return bytes(reversed(rev_res[:-1])) + eol
-        return bytes(reversed(rev_res))
+        bytes_rev_res = bytes(rev_res)[::-1]
+        if eol is None:
+            return bytes_rev_res
+        # reverse the characters in the line, except the new line character
```

Comment:
It's already reversed.

```suggestion
        # reverse the characters in the line, except the new line character
        bytes_rev_res = bytes(rev_res)[::-1]
        if eol is None:
            return bytes_rev_res
```

Reply:
Oupsi, the comment was not updated!